lib/browser/api/global-shortcut.js

```diff
@@ -1,5 +1 @@
-var globalShortcut
-
-globalShortcut = process.atomBinding('global_shortcut').globalShortcut
-
-module.exports = globalShortcut
+module.exports = process.atomBinding('global_shortcut')
```

Comment:
It is actually different case, you don't need that in `const {globalShortcut} = process.atomBinding('global_shortcut')`, but it is needed here.